lib/src/header/Header.tsx

```diff
@@ -321,7 +327,7 @@ const CloseContainer = styled.div`
   padding: ${spaces.xxsmall};
 `;
 
-const MenuContent = styled.div`
+const MenuContent = styled.div<{ backgroundType: "dark" | "light" }>`
```

Comment:
This container, and the whole layout inside the responsive menu are a bit odd, look:

![image](https://user-images.githubusercontent.com/44321109/188668721-c5d2c9f4-519a-4a23-91c0-ecffeb4bb293.png)

It has a `margin-top` of `40px` to avoid overlapping with the logo, which is quite confusing considering we control that container and it is not from the custom responsive content. I think we should refactor all the custom responsive content using flex.
Maybe the responsive logo and the close action are a row with a content justification of `space-between` and the custom content another row below.

Reply:
Done without flex.